seeder.js

```diff
@@ -2,22 +2,20 @@ import fs from 'fs/promises'// use promises version not callbacks version
 import path from 'path'
 import connect from './core/db.js'
 import { exit } from 'process'
+import { pathToFileURL } from 'url'
+import Lookup from './models/Lookup.js'
 
 const __dirname = path.resolve()
 
 connect().then(async () => {
     const files = await fs.readdir(path.join(__dirname, '/seed/'))
     for await (const file of files) {
         if (file.endsWith('.js')) {
-            // check the database if the file already seeded before
-            if(!exists) {
-                // save fileName to seeds collection, locked=true
-                // run seeder
-                // 1 minutes
-                // locked=false
+            const isExist = await Lookup.findOne().exec();
```

Comment:
```suggestion
            const isExist = await Lookup.findOne();
```
no need for `.exec()`